pptx_parser.py

```diff
@@ -0,0 +1,25 @@
+from pptx import Presentation
+
+
+def read_presentation(pptx_path):
+    """
+    Reads a PowerPoint presentation and extracts the text from each slide.
+
+    Args:
+        pptx_path (str): The path to the PowerPoint presentation file.
+
+    Returns:
+        list: A list of slide texts.
+    """
+    file = Presentation(pptx_path)
+    text = []
+    for slide in file.slides:

```

Comment:
3 for loops is too much, think of a better implementation to do so in less of a complexity.